integration/spark/src/test/scala/org/apache/spark/util/CarbonCommandSuite.scala

```diff
@@ -117,7 +117,7 @@ class CarbonCommandSuite extends QueryTest with BeforeAndAfterAll {
     val table = "carbon_table3"
     createAndLoadTestTable(table, "csv_table")
     DeleteSegmentById.main(Array(s"${location}", table, "0"))
-    CleanFiles.main(Array(s"${location}", table))
+    CleanFiles.main(Array(s"${location}", table, "false", "true", "true"))
```

Comment:
How it is passing without setting force allow carbon property

Reply:
added code to fail in the cleanFiles API also